workloads/network-perf/run.sh

```diff
@@ -36,11 +36,16 @@ if [[ ${SERVICEIP} == "true" && ${SERVICETYPE} == "metallb" ]]; then
 fi
 log "###############################################"
 
+count=0
 for pairs in ${PAIRS}; do
   export PAIRS=${pairs}
+  if [ $count -gt 0 ]; then
+    export METADATA_COLLECTION=false
+  fi
   if ! run_workload ${CR}; then
     exit 1
   fi
+  ((count++))
 done
```

Comment:
I think this way is easier (It doesn't need counter):

```suggestion
for pairs in ${PAIRS}; do
  export PAIRS=${pairs}
  if ! run_workload ${CR}; then
    exit 1
  fi
  export METADATA_COLLECTION=false
done
```

Reply:
Good call. That's what I get for putting in a PR before coffee :-)